src/_common/video/player/slider/slider.ts

```diff
@@ -27,6 +27,14 @@ export default class AppVideoPlayerSlider extends Vue {
 		thumb: HTMLDivElement;
 	};
 
+	get thumbOffset() {
```

Comment:
I think a lot of code in this file would simplify if player was a required prop.